sdk/nodejs/Makefile

```diff
@@ -18,10 +18,10 @@ include ../../build/common.mk
 export PATH:=$(shell yarn bin 2>/dev/null):$(PATH)
 
 lint::
-	./node_modules/.bin/eslint -c .eslintrc.js --ext .ts .
+	eslint -c .eslintrc.js --ext .ts .
```

Comment:
Shouldn't this be `yarn run eslint`?

Reply:
Yeah that sounds safer. The reason it works now is `export PATH:=$(shell yarn bin 2>/dev/null):$(PATH)` but might as well do `yarn run`.